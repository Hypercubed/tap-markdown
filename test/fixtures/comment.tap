TAP version 13
# sync
ok 1 should pass
    Hello,
# async
ok 2 should pass
    world!

1..2
# tests 2
# pass  2

# ok
