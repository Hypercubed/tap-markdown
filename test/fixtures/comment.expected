# Tests

- ✔ sync [pass: 1, fail: 0]
- ✔ async [pass: 1, fail: 0]

# Summary

- planned: 2
- assertions: 2
- pass: 2
- fail: 0

# Comments

## sync

```
Hello,
```

## async

```
world!
```

